CMakeLists.txt

```diff
@@ -106,6 +106,14 @@ if (NOT DEFINED COMMON_CLANG_LIBRARY_NAME)
 endif()
 set(TARGET_NAME ${COMMON_CLANG_LIBRARY_NAME}${BUILD_PLATFORM} )
 
+option(CUSTOM_PCH_EXTENSION_MAP "Set PCH extension map" OFF)
```

Comment:
CUSTOM_PCH_EXTENSION_MAP is redundant since PCH_EXTENSION is enough?

Reply:
done